GlobalConstantsService/inc/PhysicsParams.hh

```diff
@@ -109,6 +109,32 @@ namespace mu2e
     const std::vector<double>& getShankerEcoefficients() const { return _shankerEcoefficients; }
     const std::vector<double>& getShankerFcoefficients() const { return _shankerFcoefficients; }
 
+    double   getCaptureProtonRate     (targetMat material = "") const {
+      const std::string allowedMaterial = checkMaterial( material );
+      return _captureProtonRate.find(allowedMaterial)->second;
+    }
+    double   getCaptureDeuteronRate     (targetMat material = "") const {
+      const std::string allowedMaterial = checkMaterial( material );
+      return _captureDeuteronRate.find(allowedMaterial)->second;
+    }
+    double   getCaptureNeutronRate     (targetMat material = "") const {
+      const std::string allowedMaterial = checkMaterial( material );
+      return _captureNeutronRate.find(allowedMaterial)->second;
+    }
+    double   getCapturePhotonRate     (targetMat material = "") const {
+      const std::string allowedMaterial = checkMaterial( material );
+      return _capturePhotonRate.find(allowedMaterial)->second;
+    }
+
+    double   getCaptureGammaEnergy     (targetMat material = "") const {
+      const std::string allowedMaterial = checkMaterial( material );
+      return _captureGammaEnergy.find(allowedMaterial)->second;
+    }
+    double   getCaptureGammaIntensity     (targetMat material = "") const {
+      const std::string allowedMaterial = checkMaterial( material );
+      return _captureGammaIntensity.find(allowedMaterial)->second;
+    }
```

Comment:
It is confusing to have both "capture photons" and "capture gammas".  With different parameterizations (rate vs intensity) on top of that.   Do we REALLY need both?

Reply:
I will finish renaming the capture gamma to 1809keVGamma to match the renaming of the generator tool. Hopefully this will clear some confusion.

To your more fundamental question: I think there will be a use for both. I believe the calorimeter plans to see if they can detect the 1809 keV gamma and knowing its intensity is necessary to normalize. Being able to easily simulate this on top of our flat photon spectrum will be useful.

I'm happy to remove it if people want me to.

Reply:
Renaming finished in commit 1f23a940f